apps/src/netsim/netsimGlobals.js

```diff
@@ -75,6 +75,16 @@ module.exports = {
   },
 
   /**
+   * @returns {string} channels API public key for storage system
+   */
+  getChannelPublicKey: function () {
+    if (netsim_ && netsim_.environment === 'development') {
```

Comment:
The guard here makes this test-friendly (we don't fake a netsim object for all of our unit tests).